ui/src/utils/utils.tsx

```diff
@@ -14,6 +14,11 @@ export const Spinner = ({ show, style = {} }: { show: boolean; style?: React.CSS
 export enum AccessRole {
   DEFAULT_ACCESS = 'Read'
 }
+
+const getDefaultAccessRole = (): string => {
+  return window?.EPHEMERAL_ACCESS_VARS?.EPHEMERAL_ACCESS_DEFAULT_BASE_ROLE || AccessRole.DEFAULT_ACCESS;
```

Comment:
It seems that there is a conceptual problem with this change. `DEFAULT_ACCESS` is not a role per se. It is just a string to present in the UI by default. There is no associated role or any validation done by the extension. I suggest to change the implementation to make sure that this distinction is satisfied. 
Some naming options:
- `EPHEMERAL_ACCESS_DEFAULT_DISPLAY_ACCESS` instead of `EPHEMERAL_ACCESS_DEFAULT_BASE_ROLE`
- `AccessRole.DEFAULT_DISPLAY_ACCESS` instead of `AccessRole.DEFAULT_ACCESS`

Feel free to change the suggestions if you have better names :)

Reply:
Ah, that is a great point. 

I've updated it with the new names; I think the ones you presented look good.